README.md

```diff
@@ -85,932 +119,667 @@ Unlike many authorization systems where a service controls access to resources i
-    "wnfs/append": [{}]
-  },
-  "mailto:username@example.com": {
-    "msg/send": [{}],
-    "msg/receive": [
-      {
-        "max_count": 5,
-        "templates": [
-          "newsletter",
-          "marketing"
-        ]
-      }
-    ]
-  }
-}
-```
-
-## 2.6 Authority
+A resource describes the noun of a capability. The resource pointer MUST be provided in [URI] format. Arbitrary and custom URIs MAY be used, provided that the intended recipient can decode the URI. The URI is merely a unique identifier to describe the pointer to — and within — a resource.
 
-The set of capabilities delegated by a UCAN is called its "authority." This functions as a declarative description of delegated abilities.
+Having a unique agent represent a resource (and act as its manager) is RECOMMENDED. However, to help traditional ACL-based systems transition to certificate capabilities, an agent MAY manage multiple resources, and [act as the registrant in the ACL system][wrapping existing systems].
 
-Merging capability authorities MUST follow set semantics, where the result includes all capabilities from the input authorities. Since broader capabilities automatically include narrower ones, this process is always additive. Capability authorities can be combined in any order, with the result always being at least as broad as each of the original authorities.
+Unless explicitly stated, the Resource of a UCAN MUST be the Subject.
 
-``` plaintext
-                 ┌───────────────────┐  ─┐
-                 │                   │   │
-                 │                   │   │
-┌────────────────┼───┐               │   │
-│                │   │ Resource B    │   │
-│                │   │               │   │       BxZ
-│                │   │     X         │   ├─── Capability
-│     Resource A │   │               │   │
-│                │   │ Ability Z     │   │
-│         X      │   │               │   │
-│                │   │               │   │
-│     Ability Y  │   │               │   │
-│                └───┼───────────────┘  ─┘
-│                    │
-│                    │
-└────────────────────┘
-
-└──────────────────┬─────────────────┘
-                   │
-
-               AxY U BxZ
-                 authority
-```
+## Issuer & Audience
 
-The capability authority is the total rights of the authorization space down to the relevant volume of authorizations. Individual capabilities MAY overlap; the authority is the union. Except for [rights amplification], every unique delegation MUST have equal or narrower capabilities from their delegator. Inside this content space, you can draw a boundary around some resource(s) (their type, identifiers, and paths or children) and their capabilities.
+The Issuer (`iss`) and Audience (`aud`) can be conceptualized as the sender and receiver (respectively) of a postal letter. Every UCAN MUST be signed with the private key associated with the DID in the `iss` field.
```

Comment:
```suggestion
The Issuer (`iss`) and Audience (`aud`) can be conceptualized as the sender and receiver (respectively) of a postal letter. Every UCAN MUST be signed with the private key associated with the DID in the `iss` field. 

Note that the UCAN system assumes a 1:1 relationship between DID URLs and public keys, and all examples are given using `did:key`, where bare DIDs are resolvable directly and deterministically to single keys. For DID methods that support multiple keys for a given DID, verification-specific [DID URLs using fragment identifiers](https://www.w3.org/TR/did-core/#fragment) MUST be used for `iss` and `aud` values to preserve interoperability; non-key-based `verificationMethod`s MUST NOT be used, for the same reason.
```
I think I promised this as a PR over a year ago, I'm realizing as I write this up now 🙈